instrgen/driver/testdata/interface/serializer/interface.go

```diff
@@ -13,7 +13,7 @@
 // limitations under the License.
 
 //nolint:all // Linter is executed at the same time as tests which leads to race conditions and failures.
-package serializer
+package serializer	// import "go.opentelemetry.io/contrib/instrgen/testdata/interface/serializer"
```

Comment:
@pdelewski I do not understand why `instrgen_test.go` keeps replacing "space" to a "tab". Is it not a bug?

Reply:
> Regarding tests, we can improve that, however it does not seem must have for now.

The problem is that the test execution modifies the `instrgen/driver/testdata/interface/app/impl.go` and `instrgen/driver/testdata/interface/serializer/interface.go` files that you can accidently commit.

Reply:
I reverted he changes as they are not longer needed but I suggest at looking at this problem anyway 😉 https://github.com/open-telemetry/opentelemetry-go-contrib/pull/3820/commits/42181219d44cdb80e8fd1fcb2a2cbe99f5a2e829